examples/ingestron/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"encoding/json"
+	"os/signal"
+	"syscall"
+	"time"
+        r "gopkg.in/dancannon/gorethink.v2"
+	"github.com/go-steem/rpc"
+	"github.com/go-steem/rpc/transports/websocket"
+)
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatalln("Error:", err)
+	}
+}
+
+// Set the settings for the DB
+func run() (err error) {
+    Rsession, err := r.Connect(r.ConnectOpts{
+    Addresses: []string{"192.168.194.98:28015","192.168.194.91:28015","192.168.194.145:28015","192.168.194.209:28015"},
+    })
+    if err != nil {
+        log.Fatalln(err.Error())
+    }
+
+// Create a table in the DB
+var rethinkdbname string = "steemit"
+_, err = r.DBCreate(rethinkdbname).RunWrite(Rsession)
+Rsession.Use(rethinkdbname)
+	if err != nil {
+		fmt.Println("rethindb DB already made")
```

Comment:
@faddat Doesn't look like the most robust error check. Not sure how the DB driver works, but you are just assuming that getting an error here means that the DB already exists... IMO you should just fail on any error or check that the error is really that the DB already exists, if that is possible to check.